packages/Frontend/src/components/__snapshots__/Authenticate.test.jsx.snap

```diff
@@ -63,6 +63,22 @@ exports[`Authenticate should render 1`] = `
           Sign In
         </Button>
       </Link>
+      <Button
```

Comment:
Most of the changes in this snap are a result of adding a couple of buttons to the component for Facebook and Google login.